src/main/java/core/basesyntax/RightTriangleSupplier.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+public class RightTriangleSupplier {
+    private static final RandomGeometricValue randomSize = new RandomGeometricValue();
+    private static final ColorSupplier color = new ColorSupplier();
+
+    public Figure getRandomRightTriangle() {

```

Comment:
There is no validation of the parameters passed to the RightTriangle constructor. Add validation that the sides form a right triangle.

Reply:
When we are giving the sizes of two sides, I guess it is always constructable, without any extra conditions. In my opnion it will be necessary in the situation that we are obliged to declare three sides.